### PR TITLE
 Fix protoc-gen-swagger not to output gRPC method summary and descriptions as Swagger's them

### PR DESCRIPTION
**Pull request by co3k**

### CODE REVIEW

**Comment by achew22** [comment]
@co3k, thanks so much for your pull request. Looking through, I'm a bit puzzled by what it should do. It looks like the changes to the golden `.go` files all come from changes upstream. If this code path isn't exercised by our examples, do you think you could add an example that does exercise it so I can see the new functionality?

**Comment by co3k** [comment]
@achew22 
> It looks like the changes to the golden .go files all come from changes upstream

I added these changes to pass build on Travis CI. In the other pull request, I've encountered build error that is caused by the upstream changes. https://travis-ci.org/grpc-ecosystem/grpc-gateway/builds/390065893

> do you think you could add an example that does exercise it so I can see the new functionality?

You're right. I've added the https://github.com/grpc-ecosystem/grpc-gateway/pull/667/commits/ce1532e41996ea402ff4bd1529688490b245d1a3 commit to show it.

File: examples/proto/examplepb/a_bit_of_everything.proto [hunk 1]
```diff
@@ -189,6 +189,9 @@ service ABitOfEverythingService {
 		}
 	};
 
+  // Create a new ABitOfEverything
+  //
+  // This API creates a new ABitOfEverything
```
Comment (achew22) [inline examples/proto/examplepb/a_bit_of_everything.proto]: Oops, I left that comment on the commit directly -- sorry. Can you make the indentation match the rest of the file?

Comment (co3k) [reply examples/proto/examplepb/a_bit_of_everything.proto]: Oh, sorry for the mistakes. I fixed it by https://github.com/grpc-ecosystem/grpc-gateway/pull/667/commits/4b6f8895524282af806fda06e42705cbe400b817

**Comment by johanbrandhorst** [comment]
Hi! Can you please rebase this on top of master? There was recently a change to the swagger generator. You can run something like:
```
make realclean && make examples SWAGGER_CODEGEN="java -jar $HOME/local/swagger-codegen-cli.jar"
```
from the root of the repo to regenerate all examples. This will require swagger-codegen (and Java) installed.

**Comment by co3k** [comment]
@johanbrandhorst I rebased my changes and tests on Travis CI are passed.

**Comment by johanbrandhorst** [comment]
Hi, could you squash the commits as well please? Sorry I didn't specify this in the last message.

**Comment by co3k** [comment]
@johanbrandhorst OK, I've just squashed them.

**Comment by johanbrandhorst** [comment]
Thanks, I will leave it for @achew22 to approve and merge, but I'm a little confused by the PR description. Looking at the code this change looks like it parses the method comment and adds it to "Summary" and "Description" in accordance with other such comment parsing, but the PR title, description and commit message is a little confusing. Could you reword it for clarity please?

**Comment by achew22** [comment]
@johanbrandhorst another neat thing about the squash button on github is that you enter the description of the new squashed commit which makes it perfect for these situations. 2 commits squashed down into 1 and then the description hopefully clarified.

@co3k Thanks so much for your contribution! This will help in producing great documentation for APIs.

**Pull request merged**